go/cmd/vttestserver/vttestserver_test.go

```diff
@@ -213,7 +211,7 @@ func TestExternalTopoServerConsul(t *testing.T) {
 	assert.NoError(t, err)
 	defer cluster.TearDown()
 
-	assertVtGateExecute(t, cluster)
+	// assertVtGateExecute(t, cluster)
```

Comment:
I think it's fine to delete this. However, we should replace it with some other operation / command that requires the topo server to be functional.

Reply:
lemme know what you think! https://github.com/vitessio/vitess/pull/10646/commits/53c8920c45044ddd73fecd0b67625cf3600b948e